custom.js

```diff
@@ -2,6 +2,172 @@ const hamburger = document.querySelector('#nav-button');
+  },
+  {
+    name: 'Seventh Article title',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem  ',
+    image: './images/ImgPlaceholder2.png',
+    technologies: ['HTML', 'CSS', 'Bootstrap', 'JavaScript'],
+    liveversion: 'https://desorgui.github.io',
+    source: 'https://github.com/desorgui',
+  },
+];
+
+const workssection = document.querySelector('#cards-container');
+
+for (let i = 0; i < WorkData.length; i += 1) {
+  const cardDiv = document.createElement('div');
+  cardDiv.className = 'card format';
+  if (i === 0) {
+    cardDiv.classList.add('card-featured');
+    cardDiv.classList.remove('card');
+  }
+  cardDiv.innerHTML = `
+        <div class="featured-image">
+          <img
+            class="image"
+            src="${WorkData[i].image}"
+            alt="snapshot_img${i + 1}"
+          />
+        </div>`;
+  const cardContent = document.createElement('div');
+  cardContent.className = 'card-content';
+  cardDiv.appendChild(cardContent);
+  cardContent.innerHTML = `
+              <h3 class="work-title">${WorkData[i].name}</h3>
+              <p class="work-description font-inter">
+                  ${WorkData[i].description}
+              </p>
+          `;
+  const listContainer = document.createElement('ul');
+  listContainer.className = 'categories';
+  cardContent.appendChild(listContainer);
+  WorkData[i].technologies.forEach((element) => {
+    const listItem = document.createElement('li');
+    listItem.className = 'itemCat';
+    listItem.textContent = element;
+    listContainer.appendChild(listItem);
+  });
+
+  workssection.appendChild(cardDiv);
+  cardContent.innerHTML += `
+    <button class="see-project-btn seeMore font-inter">See Project</button>`;
+}
+
+const button = document.querySelectorAll('.seeMore');
+let popup = null;
+button.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    popup = document.createElement('div');
+    popup.className = 'popup';
+    popup.classList.add('popup-active');
+    document.body.appendChild(popup);
+    popup.innerHTML = `
+    <nav class="close-popup" id="close-popup">
+            <i class="fa-solid fa-xmark"></i>
+          </nav>
+          <h3 class="popup-title font-crete">
+            ${WorkData[index].name}
+          </h3>
+    `;
+    const listContainer = document.createElement('ul');
+    listContainer.className = 'categories';
+    popup.appendChild(listContainer);
+
+    WorkData[index].technologies.forEach((element) => {
+      const listItem = document.createElement('li');
+      listItem.className = 'itemCat';
+      listItem.textContent = element;
+      listContainer.appendChild(listItem);
+    });
+
+    popup.innerHTML += `
+    <div class="d-flex">
+            <div class="popup-image">
```

Comment:
- [ ] The background color of the popup window (modal) is white in the Figma design. Kindly rectify.
- [ ] In the desktop version, the height of the pop-up window is short as compared to the Figma design. As a result the project snapshot images' width Vs height proportion is distorted. Kindly rectify.

#### Design (Desktop)
![popup-window-desktop-figma](https://user-images.githubusercontent.com/44057863/169371415-abca09e7-ace6-4f94-966d-17d5efebebf3.png)
#### Impl (Desktop)
![popup-window-desktop-impl](https://user-images.githubusercontent.com/44057863/169371453-dff5b703-083e-4652-9780-41ce5ccf3434.png)

- [ ] In the mobile version, bottom part of the pop-up window (modal) is not visible. Kindly stick to the figma design.
#### design (Mobile)
![popup-window-mobil-figma](https://user-images.githubusercontent.com/44057863/169371700-6dff045c-dc0e-4032-8f1b-407733e01783.png)

#### Impl (Mobile)
![popup-window-mobil-impl](https://user-images.githubusercontent.com/44057863/169371740-3e28d7a1-44aa-4058-9035-490f4d884e32.png)